yabt/buildfile_parser_test.py

```diff
@@ -37,7 +38,10 @@ def test_parser_error(basic_conf, capsys):
         populate_targets_graph(build_context, basic_conf)
     _, err = capsys.readouterr()
     ybuild_path = join('tests', 'errors', 'parser-error', 'YBuild')
-    assert '{}", line 6'.format(ybuild_path) in err
+    if platform.python_version() >= '3.8':
```

Comment:
hell no - please don't add platform specific checks unless truly needed.

1. what changed in 3.8 that makes this move lines?
2. replace the assert with a regex check instead